drivers/scheduler/k8s/k8s.go

```diff
@@ -2847,89 +2818,51 @@ func (k *K8s) ResizeVolume(ctx *scheduler.Context, configMapName string) ([]*vol
 		}
 		if obj, ok := specObj.(*corev1.PersistentVolumeClaim); ok {
 			updatedPVC, _ := k8sCore.GetPersistentVolumeClaim(obj.Name, obj.Namespace)
-			vol, err := k.resizePVCBy1GB(ctx, updatedPVC)
+			related, err := k.relatedVolume(updatedPVC)
 			if err != nil {
 				return nil, err
 			}
-			vols = append(vols, vol)
-		} else if obj, ok := specObj.(*appsapi.StatefulSet); ok {
-			ss, err := k8sApps.GetStatefulSet(obj.Name, obj.Namespace)
-			if err != nil {
-				return nil, &scheduler.ErrFailedToResizeStorage{
-					App:   ctx.App,
-					Cause: fmt.Sprintf("Failed to get StatefulSet: %v. Err: %v", obj.Name, err),
-				}
-			}
-
-			pvcList, err := k8sApps.GetPVCsForStatefulSet(ss)
-			if err != nil || pvcList == nil {
-				return nil, &scheduler.ErrFailedToResizeStorage{
-					App:   ctx.App,
-					Cause: fmt.Sprintf("Failed to get PVC from StatefulSet: %v. Err: %v", ss.Name, err),
-				}
-			}
-
-			for _, pvc := range pvcList.Items {
-				vol, err := k.resizePVCBy1GB(ctx, &pvc)
+			if related {
```

Comment:
Do we need to check this now? Can we directly call `resizePVCBy1GB`?

Reply:
I think we do still, unless you're saying this also gets checked somewhere above us? I'm still learning my way around the code

Reply:
I see. So you want to resize the pure volume only if it is either pure_file/pure_block.